src/code/codeMaths.pug

```diff
@@ -0,0 +1,5 @@
+mixin codeMaths()
+	pre
+		| #[span.u-color-second function #[span.u-color-brand.js-info-name]](x: number) {
```

Comment:
Do you think we should add the type returned by the function?

Reply:
Ok, like I did now in 478a52b?

Reply:
👍